### ISSUE TITLE
SKR fails USB connect after sometime

### ISSUE THREAD

**Issue by yasasalwis** [bigtreetech/SKR-3#62]
All SKR boards I used failed on USB connection after using sometime. This issue present in all SKR models including SKR1.4 / PRO SKR3/PRO/EZ. No apparent cause found. No any connection whatsoever to raspberry. Thinking leaving BTT for MKS.

**Comment by MagicPiciu** [comment]
Same problem with me. Now I am looking for another board from another brand. After several searches on the Internet, I saw that it happens quite often with their motherboards.

**Comment by yasasalwis** [comment]
What is you firmware Klipper/Marlin?

**Comment by MagicPiciu** [comment]
Klipper

dum., 28 mai 2023, 11:34 Yasas Alwis ***@***.***> a scris:

**Comment by yasasalwis** [comment]
If you are using Fluidd Try Mailsail! Some are having issues with Fluidd and it's deprecated!

**Comment by MagicPiciu** [comment]
I used Fluidd, Mainsail, Octoprint. It's the same for all: 
Klipper reports: ERROR
mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

**Comment by yasasalwis** [comment]
Yes, Exactly! I'm trying to use can over USB or UART for now until a new board arrives!

**Comment by MagicPiciu** [comment]
I'm trying to connect to CAN but I can't. I don't know what else to do.
What motherboard did you order?
